### ISSUE TITLE
How to build to HTML with V2 interface?

### ISSUE THREAD

**Issue by s5bug** [tectonic-typesetting/tectonic#774]
`--outfmt` with the V1 interface supports building HTML. Can the V2 interface do this?

Alternatively, how can I build a V2 project with the V1 interface so I can use `--outfmt`?

**Comment by pkgw** [comment]
Indeed, the V2 interface doesn't currently support the HTML output mode. This is only because the current HTML output is primitive enough that I didn't think that it's been worthwhile to wire up at the moment. If you're using HTML with the V1 interface and would like to see that in V2, I can certainly add it!

**Comment by s5bug** [comment]
I haven't used the V1 interface actually. I had to write a final paper for a class where the grading tools only work with `.docx`, and the only citation system I was comfortable with at the time was BibTex. I've used Tectonic before where I could turn in PDFs, but the recommended way to get LaTeX into a .docx format is to go from LaTeX -> HTML -> .docx.

I eventually was able to figure out OpenOffice's citation system, and the semester is long-over. Should I leave this issue open as a tracking issue?
